mono_repo/test/root_config_test.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
+
+import 'package:mono_repo/src/commands/github/generate.dart';
+import 'package:mono_repo/src/root_config.dart';
+import 'package:path/path.dart' as path;
+import 'package:test/test.dart';
+
+void main() {
+  group('RootConfig', () {
+    test('parseActionVersions', () {
```

Comment:
it would be good to also have a check that `mono_repo validate` can pass regardless of the version that is in these files